SwiftSignalRClient.podspec

```diff
@@ -1,13 +1,13 @@
 Pod::Spec.new do |s|
   s.name                   = "SwiftSignalRClient"
-  s.version                = "0.4.1"
+  s.version                = "0.4.2"
   s.summary                = "Swift SignalR Client for the ASP.Net Core version of SignalR."
   s.homepage               = "https://github.com/moozzyk/SignalR-Client-Swift"
   s.license                = { :type => "Attribution License", :file => "LICENSE" }
-  s.source                 = { :git => "https://github.com/moozzyk/SignalR-Client-Swift.git", :tag => "0.4.1" }
+  s.source                 = { :git => "https://github.com/moozzyk/SignalR-Client-Swift.git", :tag => "0.4.2" }
   s.authors                = { "Pawel Kadluczka" => "moozzyk@gmail.com" }
   s.social_media_url       = "https://twitter.com/moozzyk"
-  s.swift_version          = "4.2"
+  s.swift_version          = "5.0"
```

Comment:
Note to myself - pushing with 1.6.1 results in
```
[!] The Pod Specification did not pass validation.
The following validation failed:
- Warnings: Unrecognized `swift_version` key.
```
A corresponding inssue: https://github.com/CocoaPods/CocoaPods/issues/8635